src/JestProcessManagement/JestProcess.ts

```diff
@@ -0,0 +1,80 @@
+import { Runner, ProjectWorkspace } from 'jest-editor-support'
+
+export class JestProcess {
+  static readonly keepAliveLimit = 5
+  private runner: Runner
+  private projectWorkspace: ProjectWorkspace
+  private onExitCallback: Function
+  private jestSupportEvents: Map<string, (...args: any[]) => void>
+  private keepAliveCounter: number
+  public keepAlive: boolean
+  public watchMode: boolean
+  public stopRequested: boolean
+
+  private startRunner() {
+    this.stopRequested = false
+    let exited = false
+    this.runner = new Runner(this.projectWorkspace)
```

Comment:
It would be useful to create the Runner with the shell option enabled (requires jest-editor-support 22.1.3) at least on Windows in order to fix #98.

Reply:
Hi @stephtr. Thanks for pointing this out. I will take a look. Hopefully, we all got some slack very soon to start review.